packages/database/tests/pluto.test.ts

```diff
@@ -13,9 +13,9 @@ import {
 } from "@atala/prism-wallet-sdk";
 import * as sinon from "sinon";
 import { RxStorage } from "rxdb";
-import InMemory from "../../inmemory";
-import IndexDb from "../../indexdb";
-import { createLevelDBStorage } from '../../leveldb'
+import InMemory from "../../inmemory/src";
+import IndexDb from "../../indexdb/src";
+import { createLevelDBStorage } from '../../leveldb/src'
```

Comment:
note with vitest we can pull the `.ts` in directly ... it rebuilds everything on the fly hella fast